src/codeu/chat/client/commandline/Panel.java

```diff
@@ -30,7 +33,46 @@
     void invoke(List<String> args);
   }
 
+  class TimerCommand {
```

Comment:
Kinda done. All the inheritance from Command was getting messy. I didn't want the Action interface to have the List<String> args in the parameter since there is no way to pass those arguments to the scheduler, so I simply created a different interface that should be used for the Scheduled events